src/msg/async/dpdk/DPDK.h

```diff
@@ -44,6 +44,7 @@
 #include "net.h"
 #include "toeplitz.h"
 
+using namespace std;
```

Comment:
could you avoid `using namespace std` in header files? the idea is to avoid polluting the global naming space and give the .cc files including this header file the illusion that `std` namespace is always "used".